public/css/components.css

```
@@ -1441,6 +1441,59 @@ body[data-theme="dark"] .hero-primary-action:disabled {
   transform: translateY(0);
   opacity: 1;
 }
+
+/* Threshold → dialog FLIP morph. Calm settle with --ease-standard, not a
+   spring overshoot — overshoot is reserved for solidified-state celebration
+   per the motion spec. */
+@keyframes thresholdMorphIn {
```

Stylelint (17.10.0)

[error] Expected keyframe name "thresholdMorphIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)